infra/base-images/base-builder/Dockerfile

```diff
@@ -142,6 +142,7 @@ RUN cd $SRC && \
 COPY precompile_afl /usr/local/bin/
 RUN precompile_afl
 
+RUN apt-get update -y
```

Comment:
You don't want to do this. Because of caching this step can cause the build to fail. You should put the apt-get update in the same step where it is needed.